pallets/gear/src/tests.rs

```diff
@@ -8635,39 +8627,51 @@ mod utils {
         assert_eq!(status, DispatchStatus::Success)
     }
 
-    #[track_caller]
-    pub(super) fn get_last_event_error(message_id: MessageId) -> String {
+    fn get_last_event_error_and_status_code(message_id: MessageId) -> (String, StatusCode) {
```

Comment:
track_caller

Reply:
we don't need track caller for not public functions